README.md

```diff
@@ -0,0 +1,174 @@
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **SPA functionality with React**
+- **Handle state management with Redux**
+- **Integrate Rails with React using Api controllers**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo (not available) <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- <a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby</a>
+- <a href="https://rubyonrails.org/">Ruby on Rails</a>
+- <a href="https://www.postgresql.org/">PostgreSQL</a>
+- any code editor
+
+### Setup
+
+To setup the project follow the steps:
+
+1. Clone this project using Git Bash:
+
+   ```
+   git clone git@github.com:Mujeeb4582/hello-rails-react.git
+   ```
+
+2. Inside the project directory, install the project's dependencies
+
+   ```
+   bundle install
+   ```
+
+3. Setup the app's database
+   ```
+   rails db:create
+   rails db:migrate
+   rails db:seed
+   ```
+
+### Available Scripts
+
+You can run the project by executing the following command:
+
+- ```
+  rails server
+  or
+  bin/dev
+  ```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mujeeb ur Rahman**
+
+- GitHub: [![Mujeeb](https://img.shields.io/badge/-Mujeeb_Ur_Rahman-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/Mujeeb4582)
+- Twitter: [![mujeeburrahman](https://img.shields.io/badge/-mujeeburahman-blue?logo=Twitter&logoColor=skyBlue&style=plastic)](https://twitter.com/Mujeebu93992980)
+- LinkedIn: [![Mujeeb-ur-Rahman](https://img.shields.io/badge/-Mujeeb_Ur_Rahman-blue?logo=LinkedIn&logoColor=#0072b1&style=plastic)](https://www.linkedin.com/in/rahman-mujeeb)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add authentication using JWT**
+- [ ] **Add authorization in react components**
+- [ ] **Add styles to components**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Put a ⭐️ if you like this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for project inspiration.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Hi @Mujeeb4582 
You have done a great job so far 

- [ ] Please fix the License link on your README file, the current link is broken.